staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit.go

```diff
@@ -140,7 +140,7 @@ func WithAudit(handler http.Handler, requestContextMapper request.RequestContext
 
 func auditStringSlice(inList []string) string {
 	if len(inList) == 0 {
-		return ""
+		return "<none>"
```

Comment:
This entire check is unnecessary from what I see. Everywhere we call `auditStringSlice` we check the length of that `inList` being `> 0`. This is because we set different "empty" term for each of those groups.